integration-cli/docker_cli_run_test.go

```diff
@@ -4849,3 +4850,34 @@ func (s *DockerSuite) TestRunEmptyEnv(c *check.C) {
 	c.Assert(err, checker.NotNil)
 	c.Assert(out, checker.Contains, expectedOutput)
 }
+
+// #28658
+func (s *DockerSuite) TestSlowStdinClosing(c *check.C) {
+	testRequires(c, DaemonIsLinux)
```

Comment:
IIUC, you can just use the busybox cat here in Windows
https://github.com/docker/docker/blob/420e22fad4dcdff3b8025666216cc85fc9358f69/integration-cli/docker_cli_inspect_test.go#L145